Resources/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "fos-router",
+  "version": "1.6.0",
```

Comment:
This should be updated to reflect the version number in the rest of the bundle

Reply:
Hmmm... that was correct at the time of forking - the world of FOSJsRouting is moving quickly at present :-)

I've updated it to 2.2 - see if I can get the other feedback on this PR addressed before it's too late to be that version!